cmd/kubectl-testkube/commands/dashboard.go

```diff
@@ -74,6 +93,26 @@ func NewDashboardCmd() *cobra.Command {
 	return cmd
 }
 
+func readinessCheck(apiURI, dashboardURI string) (bool, error) {
+	ui.Info("waiting for port-forwarded services ...")
+	for {
+		time.Sleep(500 * time.Millisecond)
+
+		apiResp, err := http.Get(apiURI + "/info")
+		if err != nil {
+			ui.ExitOnError("getting status from api ")
```

Comment:
defer will not be called on os.Exit, so kubectl commands might become Zombie processes in this case

Reply:
yeah - good point - they are not - trying to make this works ok